File: src/test/java/net/slipp/support/utils/RandomStringUtilsTest.java
package net.slipp.support.utils;

import org.apache.commons.lang.RandomStringUtils;
import org.junit.Test;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class RandomStringUtilsTest {
    private static Logger log = LoggerFactory.getLogger(RandomStringUtilsTest.class);

    @Test
    public void generateRandomPassword() throws Exception {
        log.debug("password : {}", RandomStringUtils.randomAlphanumeric(12));
    }
}

File: src/test/scala/net/slipp/support/utils/RandomStringUtilsTest.scala
package net.slipp.support.utils

import org.apache.commons.lang3.RandomStringUtils
import org.junit.Test
import org.slf4j.Logger
import org.slf4j.LoggerFactory

class RandomStringUtilsTest {
  private val log: Logger = LoggerFactory.getLogger(classOf[RandomStringUtilsTest])

  @Test
  def generateRandomPassword {
    log.debug("password : {}", RandomStringUtils.randomAlphanumeric(12))
  }
}
